engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -69,18 +79,7 @@ public void initCounters() {
 
     @CacheResult(cacheName = "rbac-recipient-users-provider-get-users")
     public List<User> getUsers(String accountId, boolean adminsOnly) {
-        Timer.Sample getUsersTotalTimer = Timer.start(meterRegistry);
-        List<User> users = getWithPagination(
-            page -> {
-                Timer.Sample getUsersPageTimer = Timer.start(meterRegistry);
-                Page<RbacUser> rbacUsers = retryOnError(() ->
-                        rbacServiceToService.getUsers(accountId, adminsOnly, page * rbacElementsPerPage, rbacElementsPerPage));
-                getUsersPageTimer.stop(meterRegistry.timer("rbac.get-users.page", "accountId", accountId));
-                return rbacUsers;
-            }
-        );
-        getUsersTotalTimer.stop(meterRegistry.timer("rbac.get-users.total", "accountId", accountId, "users", String.valueOf(users.size())));
-        return users;
+        return transformToUser(itUserService.getUsers(accountId, adminsOnly), adminsOnly);
```

Comment:
Wouldn't it be safer to start with the two implementations (RBAC and IT) side by side and switch from one to the other with a boolean from the configuration, instead of removing the old way already?